tasks/extract-binaries-from-image/extract-binaries-from-image.yaml

```diff
@@ -0,0 +1,74 @@
+---
+apiVersion: tekton.dev/v1
+kind: Task
+metadata:
+  name: extract-binaries-from-image
+  labels:
+    app.kubernetes.io/version: "0.1.0"
+  annotations:
+    tekton.dev/pipelines.minVersion: "0.12.1"
+    tekton.dev/tags: release
+spec:
+  description: >-
+    Tekton task that extracts binaries to be released on github.com from an image
+  params:
+    - name: image_binaries_path
+      type: string
+      description: Path inside the image where the binaries to extract are stored
+      default: "releases"
+    - name: binaries_dir
```

Comment:
I find it confusing to have the same name for a parameter and a result. And each of them is different - this one is without the subdirectory and the result is including the subdirectory. Could we remove this parameter and simply generate a directory? It can even be a hardcoded name, since it's inside the "random" subdirectory.

Reply:
I updated the task. Not asking for the parameter anymore and not giving any result. The way it worked before was because the result was storing the entire path, but for consistency with other tasks, I thought it was better to use the subdirectory. So it makes sense now to simply hard code the directory

Reply:
When you say:

>  the result was storing the entire path, but for consistency with other tasks, I thought it was better to use the subdirectory

Which task stores the path without the subdirectory? For example here we store paths in the result including the subdirectory: https://github.com/redhat-appstudio/release-service-catalog/blob/main/tasks/collect-data/collect-data.yaml#L73

And I think that's what we try to stick to - so that the path where something is saved is defined in one place and the consumer will take it from the result. And relying on the results creates a logical flow of the tasks, so then you don't have to explicitly say which task depends on which other task - if it depends on a result, Tekton will take care of the order. Or maybe there's some other reason to do it this way? WDYT?

Reply:
@mmalina I think you are totally right. When I was testing the task I changed a lot of things around that were not working and lost the initial point of the result and didn't notice what you say here! I'll update it and let you know

Reply:
@mmalina done!

Reply:
Cool!